packages/zoe/src/contracts/newSwap/collectFees.js

```diff
@@ -0,0 +1,28 @@
+// @ts-check
+
+import { amountMath } from '@agoric/ertp';
+import { trade } from '../../contractSupport';
+
+export const makeMakeCollectFeesInvitation = (zcf, feeSeat, centralBrand) => {
+  const collectFees = seat => {
+    const allocation = feeSeat.getAmountAllocated('RUN', centralBrand);
+    trade(
```

Comment:
Seems like this would be better using a simple `reallocate` rather than `trade` helper.

Reply:
It is more direct that way, and eslint formats it more compactly.  done.